app.ts

```diff
@@ -314,7 +318,7 @@ async function setupWebPackDevMiddleware(router) {
     const {default: webpack} = await import('webpack');
     /* eslint-enable */
 
-    const webpackCompiler = webpack(webpackConfig);
+    const webpackCompiler = webpack(webpackConfig as any);
```

Comment:
Maybe we can assert it to the webpack config type?

Reply:
Ah, I had tried but failed last night but just got it working, thanks for bringing it up again! Because of the `const {default: webpack} = await import('webpack');` we can't do `webpack.Configuration` but we can get it via `ElementType<Parameters<typeof webpack>[0]>`. Not sure if this is the easiest solution but it's all I can think of :)